Resources/Prototypes/Nyanotrasen/Catalog/Fills/Vending/Inventories/repdrobe.yml

```diff
@@ -0,0 +1,22 @@
+- type: vendingMachineInventory
+  id: RepDrobeInventory
+  name: RepDrobe
+  spriteName: repdrobe
+  startingInventory:
+    ClothingUniformJumpsuitReporter: 2
+    ClothingUniformJumpsuitDetective: 2
+    ClothingUniformJumpskirtDetective: 2
+    ClothingShoesColorBrown: 2
+    ClothingShoesColorBlack: 2
+    ClothingShoesColorWhite: 2
+    ClothingOuterCoatDetective: 2
+    ClothingOuterVestDetective: 2
+    ClothingHeadHatFedoraBrown: 2
+    ClothingUniformJumpsuitDetectiveGrey: 2
+    ClothingUniformJumpskirtDetectiveGrey: 2
+    ClothingNeckTieDet: 2
+    ClothingHeadHatFedoraGrey: 2
+    ClothingHandsGlovesColorBlack: 2
+    DrinkDetFlask: 2
```

Comment:
But they're so prettyyyy x(
Alright, you want the trench coat gone, and the flask gone (because no one is allowed to have drinking utilities >:T), everything else is fine?

Reply:
yeah

Reply:
Donezo